client/src/components/QA/AddingForm.jsx

```diff
@@ -126,18 +127,20 @@ class AddingForm extends React.Component {
 
 
               <tr>
-                <td>Email:</td>
+                <td >Email:</td>
                 <td>
-                  <input maxlength='60'
+                  <input
+                    className='qa-input-email'
+                    maxlength='60'
                     type='email'
-                    placeholder='Why did you like the product or not?'
+                    placeholder='Example: jack@email.com'
                     id='email'
                     value={this.state.email}
                     onChange={this.handleOnChange.bind(this)}>
```

Comment:
This instantiates a new function with every render. Please bind functions in the constructor.